### PR TITLE
Fix: Enrichers should resolve relative paths against project directory, not working directory

### PR DESCRIPTION
**Pull request by erikgb**
I am having issues using the ConfigMapEnricher in multi-module Maven projects. The enricher resolves files added to ConfigMap relative to the **working directory**, and not the **project directory**. This PR fixes this issue.

### CODE REVIEW

**Comment by rohanKanojia** [comment]
@erikgb : Thanks for the PR :+1: . Could you please add a changelog entry regarding this change? Could it be that few other enrichers are also affected with this? I see one `FileDataSecretEnricher` also accessing file like this one.

**Comment by erikgb** [comment]
@rohanKanojia : If there are other enrichers with the same bug, we should fix it now! Copy-modify is the most commonly used "pattern" in software development, so..... What do you think about introducing a reusable method on MavenEnricherContext to resolve Maven paths - paths relative to the project directory (project.basedir Maven property)?

**Comment by rohanKanojia** [comment]
Thanks, Sounds good to me.

**Comment by erikgb** [comment]
@rohanKanojia : I have now added a changelog entry. I also had to refactor one test to a parameterized test. It just became too messy when adding more expectations. I hope you like it, but feel free to rebase at will - and eventually skip commits. At the end, I think everything should be squashed anyway. Let me know if you want me to do it....

File: enricher/standard/src/test/java/io/fabric8/maven/enricher/standard/ConfigMapEnricherTest.java [hunk 1]
```diff
@@ -27,55 +27,64 @@
...
+import org.junit.runners.Parameterized;
 
 import java.io.IOException;
-import java.net.URISyntaxException;
 import java.nio.file.Files;
+import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```
Comment (rohanKanojia) [inline enricher/standard/src/test/java/io/fabric8/maven/enricher/standard/ConfigMapEnricherTest.java]: nit, avoid wildcard imports.

Comment (erikgb) [reply enricher/standard/src/test/java/io/fabric8/maven/enricher/standard/ConfigMapEnricherTest.java]: Of course! I have to fix the code formatting template in IDEA on my home laptop. Fixed now.

**Pull request merged**